src/plugins/lua/luaplugin.h

```diff
@@ -45,23 +47,12 @@ class LuaTableWriter;
 /**
  * This plugin allows exporting maps as Lua files.
  */
-class LUASHARED_EXPORT LuaPlugin : public Tiled::WritableMapFormat
+class LuaUtilWriter
```

Comment:
I would rename this class to just `LuaWriter`.

Reply:
Done